RUFAS/routines/animal/ration/ration_driver.py

```diff
@@ -481,7 +482,7 @@ def report_ration(  # noqa
                 if nutr == "DM":
                     if available_feeds[key][nutr]:
                         nutrient_amount["as_fed"] += val / (
-                            available_feeds[key][nutr] / 100
+                            available_feeds[key][nutr] / GeneralConstants.FRACTION_TO_PERCENTAGE
```

Comment:
```suggestion
                            available_feeds[key][nutr] * GeneralConstants.PERCENTAGE_TO_FRACTION
```
The conversions in General Constants have been written so that they can always be multiplied - which doesn't have any meaningful impact on the result, but is a little easier to read/understand.